config/locales/en.yml

```diff
@@ -54,6 +54,7 @@ en:
       profile_not_found: >
         You are using a non-existent profile called '%{profile}',
         please check your technologies.
+      invalid_technologies: "The LES contains the following invalid technologies: %{invalid_technologies}. This will result in an incomplete network calculation."
```

Comment:
> and show the full error message on the edit page?

I like that! We could even color the offending technologies red to make them stand out.